ansible/default-playbook.yml

```diff
@@ -8,7 +8,7 @@
        - "packer"
 
    roles:
-      - ansible-role-pcp
+      #- ansible-role-pcp
```

Comment:
Why you don't remove this line instead of comment?

Reply:
We could remove "ansible-role-pcp" from "packer-images/ansible/requirements.yml" so as not to install the role.

Reply:
Fixed! Thanks!